Dockerfile.foreman

```diff
@@ -0,0 +1,18 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV CELERY_VERSION 4.0.2
+
+COPY workers/requirements.in .
+
+RUN pip install pip-tools && \
+    pip-compile requirements.in && \
+    pip install -r requirements.txt
```

Comment:
Nice catch. I think it's important to capture version numbers in source control.